tools/who_what_benchmark/whowhatbench/text2image_evaluator.py

```diff
@@ -115,7 +117,7 @@ def worst_examples(self, top_k: int = 5, metric="similarity"):
         return res
 
     def _generate_data(self, model, gen_image_fn=None, image_dir="reference"):
-        def default_gen_image_fn(model, prompt, num_inference_steps, generator=None):
+        def default_gen_image_fn(model, prompt, num_inference_steps, *, generator=None, **kwargs):
```

Comment:
`*` - why is it needed ?
I think you can left just **kwargs and specify {'empty_adapters': self.empty_adapters} on str 167 or add empty_adapters=False here